buildSrc/src/main/kotlin/Extensions.kt

```diff
@@ -120,6 +131,23 @@ fun Project.setupLibraryModule(block: LibraryExtension.() -> Unit = {}): Library
     }
 }
 
+fun Project.setupAppModule(block: BaseAppModuleExtension.() -> Unit = {}): BaseAppModuleExtension {
+    return (setupBaseModule() as BaseAppModuleExtension).apply {
+        defaultConfig {
+            versionCode = project.versionCode
+            versionName = project.versionName
+            resConfigs("en")
+            multiDexEnabled = true
```

Comment:
I don't think `multiDexEnabled = true` should be part of this base function as it may not be required for all app modules in the future.

Reply:
Separated it from `setupAppModule`